src/environmentd/src/environmentd/main.rs

```diff
@@ -209,7 +209,12 @@ pub struct Args {
     /// Frontegg arguments.
     #[clap(flatten)]
     frontegg: FronteggCliArgs,
-
+    /// Self hosted auth
+    #[clap(long, env = "ENABLE_SELF_HOSTED_AUTH")]
+    enable_self_hosted_auth: bool,
+    /// Self hosted auth over internal port
+    #[clap(long, env = "ENABLE_SELF_HOSTED_AUTH_INTERNAL")]
+    enable_self_hosted_auth_internal: bool,
```

Comment:
Ignore me if this was part of a design with the Cloud team, but I'm not sure how these are intended to be used? How do these relate with the `enable_self_managed_auth` dyncfg?

Reply:
Probably gonna punt this to a follow up PR